docker-gc.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Docker-gc Service
+After=docker.service
+Requires=docker.service
+[Service]
+Restart=always
+ExecStartPre=-/usr/bin/docker kill docker-gc
+ExecStartPre=-/usr/bin/docker rm docker-gc
+ExecStartPre=/usr/bin/docker pull ndeloof/docker-gc:latest
```

Comment:
I'm cheating in fact. CoreOS is all about reboot for self-upgrading. So it's pulling at each reboot the latest image ;-)